dwl.c

```diff
@@ -686,7 +740,7 @@ focusclient(Client *c, struct wlr_surface *surface, int lift)
 		else
 			wlr_xdg_toplevel_set_activated(ptl->xdg_surface, 0);
 	}
-	if (tl != ptl && tl) {
+	if (tl) {
```

Comment:
xwayland surfaces are not always activated upon creation. This seems a simple place to ensure it is so. Happy to refactor if there's a better place.

Reply:
Seems legit.  We could even have simplified this condition before adding xwayland, thanks to wlroots' selective configure events.